tools/submit.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+if [[ $# < 1 ]]
+then
+    echo "Usage: ./submit.sh JOB, where JOB is the job script."
+    exit 1
+fi
+
+ERR=0
+
+BIN=$(sed -n 's/BIN\s*=\s*//p' < $1)
+CFG=$(sed -n 's/CFG\s*=\s*//p' < $1)
+
+if ! [[ -f $BIN ]]
+then
+   echo "Binary $BIN not found!"
+   exit 1
+fi
+
+if ! [[ -f $BIN ]]
+then
+   echo "Config $CFG not found!"
+   exit 1
+fi
+
+./check_vlasiator_cfg.sh $BIN $CFG > check.txt
+
+if grep -q "Invalid options" check.txt 
+then
+    echo "Invalid options in config file, check check.txt"
+    echo ""
+    ERR=1
+fi
+
+SW=$(sed -n s/^file_..\\s*=\\s*//p < $CFG)
```

Comment:
So does this check for several sw files? e.g. if we use multipop or have numerous different inflow boundaries.

Reply:
Fixed to iterate over every sw file